Examples/MagicWeather/MagicWeather/Sources/Controllers/PaywallViewController.swift

```diff
@@ -59,15 +59,15 @@ class PaywallViewController: UITableViewController {
         
         /// - Configure the PackageCell to display the appropriate name, pricing, and terms
         if let package = self.offering?.availablePackages[indexPath.row] {
-            cell.packageTitleLabel.text = package.product.localizedTitle
+            cell.packageTitleLabel.text = package.storeProduct.localizedTitle
             cell.packagePriceLabel.text = package.localizedPriceString
             
-            if let intro = package.product.introductoryPrice {
-                if intro.price == 0 {
-                    cell.packageTermsLabel.text = "\(intro.subscriptionPeriod.periodTitle()) free trial"
-                } else {
-                    cell.packageTermsLabel.text = "\(package.localizedIntroductoryPriceString) for \(intro.subscriptionPeriod.periodTitle())"
-                }
+            if let intro = package.storeProduct.introductoryDiscount {
+                let packageTermsLabelText = intro.price == 0
+                ? "\(intro.subscriptionPeriod.periodTitle()) free trial"
+                : "\(package.localizedIntroductoryPriceString ?? "unknown") for \(intro.subscriptionPeriod.periodTitle())"
```

Comment:
I would force-unwrap `localizedIntroductoryPriceString`, this is already checking that `StoreProduct.introductoryDiscount` is not `nil`, so per the docs, this can't be `nil`
Also looks like the docs of that method need to be updated to reference `introductoryDiscount` instead of `introductoryPrice`.

Reply:
good call, updated